League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -457,5 +457,15 @@ private void BrowseButton_Click(object sender, EventArgs e)
                     installDirectoryText.Text = selectPath.SelectedPath;
             }
         }
+
+        private void cloningProgressLabel_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void downloadingProgressLabel_Click(object sender, EventArgs e)
```

Comment:
Delete this and its corresponding event. If you think that these are important please add `TODO:` that explains what is planned here.